public/content/en/vibed/deploy-on-heroku.md

```diff
@@ -0,0 +1,188 @@
+# Deploy on Heroku
+
+### Pre-requirements
+
+- Heroku [account]((https://signup.heroku.com/login))
+- [Git](https://git-scm.com/) installed
+- Your application should compile without error using dmd (`dub build --build=release`). 
+
+### 1 : Setup the app
+
+The first thing to do before deploying is binding the app's port to heroku's one. 
```

Comment:
How about?

Heroku needs to know how to communicate with a deployed application. Hence a global `PORT` environment variable is provided which needs to be injected into an application and it should bind and listen to this port. For development a default port (here 8080) should be set: